service/snap/snap.go

```diff
@@ -390,7 +322,8 @@ func (s Service) ConfigOverride() error {
 	if len(s.conf.Limit) == 0 {
 		return nil
 	}
-	for _, backgroundService := range s.app.BackgroundServices {
+
+	for _, backgroundService := range s.app.BackgroundServices() {
 		unitOptions := systemd.ServiceLimits(s.conf)
```

Comment:
Irrelevant to the changeset but why aren't these 3 lines pulled out of the loop?

Reply:
Ouch, missed that one...